### ISSUE TITLE
Consider not using `git+https`

### ISSUE THREAD

**Issue by jesperronn** [apiaryio/gavel.js#101]
Commit https://github.com/apiaryio/gavel.js/commit/68c08ea33b58002962f0d34a0eec10149bd3f9d0 introduced fetching a npm package via `https+git`, which servers with older versions of git do not support.

see comment https://github.com/apiaryio/gavel.js/commit/68c08ea33b58002962f0d34a0eec10149bd3f9d0#commitcomment-29333973

For machines not able to update Git, this change blocks usage of Gavel and who depend on it. Most notably Dredd

I found this problem on a machine with Git 2.1.2:

```
$ git --version
git version 2.1.2
$ git ls-remote -h -t https://git@github.com/josdejong/jsonlint.git
fatal: Unable to find remote helper for 'https'
```

Question is if you can see problems by reverting that line of code to the old version:

Suggestion:
```diff
-    "jsonlint": "git+https://git@github.com/josdejong/jsonlint.git",
+    "jsonlint": "josdejong/jsonlint",	
```

I would think this will preserve functionality as-is

**Comment by jesperronn** [comment]
I know other alternatives have been discussed, but had other problems, but of course it would be better to use a json parser which is maintained, but I will leave that for other PRs to discuss

**Comment by honzajavorek** [comment]
Since the change was useless - https://github.com/apiaryio/gavel.js/commit/68c08ea33b58002962f0d34a0eec10149bd3f9d0#commitcomment-22365106 - I'm willing to change it back to `josdejong/jsonlint` or just anything equivalent. Would you test whether it helps and send a PR?

**Comment by jesperronn** [comment]
I'll make a PR, then. 

How to test, I am not sure. I can see that the specific build server started failing after a code version update.

So I know the old version works on that Git version.

A test case could be to make a package.json with the `jsonlint` dependency locally on that server and run `npm install`?

**Comment by honzajavorek** [comment]
Yes, I think that should be sufficient to verify.

**Comment by honzajavorek** [comment]
Thank you! Solved in #102.
